GoogleCloudPlatform/anthos-service-mesh-samples
# Correct Istio-to-ASM VirtualService comment

### Background 
* The comment inside `/docs/migrate-to-managed-asm/virtual-service.yaml` is incorrect.

### Change Summary
* Update the explanation with a _correct_ explanation.

### Testing Procedure
* See how the VirtualService sample is being used in the [associated tutorial](https://cloud.google.com/service-mesh/docs/migrate-istio-to-anthos-service-mesh#test-using-canary-deployment).
* Make sure the new explanation is correct.
